pkgs/os-specific/linux/aic8800/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  kernel,
+  kernelModuleMakeFlags,
+  dpkg,
+  aic8800-firmware,
+}:
+stdenv.mkDerivation (finalAttr: {
+  name = "aic8800";
+  version = aic8800-firmware.version;
+  src = aic8800-firmware.src;
+
+  hardeningDisable = [ "pic" ];
+
+  nativeBuildInputs = kernel.moduleBuildDependencies ++ [ dpkg ];
+
+  makeFlags = kernelModuleMakeFlags ++ [
+    "KDIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
+  ];
+
+  patchPhase = ''
+    runHook prePatch
+
+    dpkg-source --before-build .
+    cp -fv ${./Makefile} ./Makefile
```

Comment:
Why can't we use the upstream Makefiles?

Reply:
But that's just the PCIe variant of the driver. There are several other variants scattered in different folders, so a custom Makefile to tie them up should be fine.

Reply:
I don't know how to build it in a single, parallelized make invocation[1] without maintaining a custom Makefile. Just revert this part back to the first revison.

1. https://github.com/NixOS/nixpkgs/pull/443520#discussion_r2353340025

Reply:
Oh, sorry, I think I misread when I made that initial comment, and assumed that it would be easy to combine.  I think it would be better to do simpler and slower than to have a custom Makefile in Nixpkgs. Sorry for the miscommunication. :(